LiskHQ/lisk-nano
# Open external URLs on openUrl event - Closes #787

The change in this PR add a listener to openUrl event transmitted from main process.

**How to test:**
 - run `npm run build && npm run start`
 - Create links to different routes of our React app, they should activate the electron app and navigate to that particular link.
   - `lisk://main/voting/send` : should open the send modal in voting tab
   - `lisk://main/wrong-tab` : should show a toaster informing about wrong URL
   - `lisk://main/voting/wrong-dialog` : open voting tab without any modals
   - `lisk://register` : should open the register modal in login page
and so on.

Closes #787
